app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java

```diff
@@ -74,7 +74,7 @@ public static void deleteAll(Context c) {
 
 	public List<Measurement> getMeasurements() {
 		if (measurements == null)
-			measurements = SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false), Measurement_Table.is_done.eq(true)).orderBy(Measurement_Table.is_anomaly, false).orderBy(Measurement_Table.is_failed, false).orderBy(Measurement_Table.id, false).queryList();
+			measurements = SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false), Measurement_Table.is_done.eq(true)).orderBy(Measurement_Table.is_anomaly, false).orderBy(Measurement_Table.is_failed, false).orderBy(Measurement_Table.id, true).queryList();
```

Comment:
I don't understand this change

Reply:
The order inside the test detail result screen is gonna be ascending, not descending, so first tests first